tests/test_image_utils.py

```
@@ -16,10 +16,20 @@ def test_calculate_subsample(self):
+            calculate_subsample(100, 100, 0, 100)
+        with self.assertRaisesRegex(ValueError, "max_width and max_height must be positive"):
+            calculate_subsample(100, 100, 100, -5)
 
-    def test_file_to_base64(self):
+    def test_file_to_base64_not_found(self):
+        # ファイルが存在しない場合、FileNotFoundError が発生することを確認
+        with self.assertRaises(FileNotFoundError):
+            file_to_base64("non_existent.png")
+
+    @patch("os.path.exists", return_value=True)
+    def test_file_to_base64(self, mock_exists):
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_exists`

(ARG002)